src/GitHub/Endpoints/Organizations/Teams.hs

```diff
@@ -139,6 +142,24 @@ listTeamMembersR tid r = PagedQuery ["teams", toPathPart tid, "members"] [("role
         TeamMemberRoleMaintainer  -> "maintainer"
         TeamMemberRoleMember      -> "member"
 
+-- | The repositories of a single team, by team id.
+-- | With authentication
+--
+-- > listTeamRepos' (Just $ OAuth "token") 1010101
+listTeamRepos' :: Maybe Auth -> Id Team -> IO (Either Error (Vector Repo))
+listTeamRepos' auth tid = executeRequestMaybe auth $ listTeamReposR tid Nothing
+
+-- | Query team repositories
```

Comment:
A ful-stop after the title. (Next line is rendered next to it, check `stack haddock`)